app/main.py

```diff
@@ -1,52 +1,33 @@
-import json
-import xml.etree.ElementTree as ET
-
-
-class Book:
-    def __init__(self, title: str, content: str):
-        self.title = title
-        self.content = content
-
-    def display(self, display_type: str) -> None:
-        if display_type == "console":
-            print(self.content)
-        elif display_type == "reverse":
-            print(self.content[::-1])
-        else:
-            raise ValueError(f"Unknown display type: {display_type}")
-
-    def print_book(self, print_type: str) -> None:
-        if print_type == "console":
-            print(f"Printing the book: {self.title}...")
-            print(self.content)
-        elif print_type == "reverse":
-            print(f"Printing the book in reverse: {self.title}...")
-            print(self.content[::-1])
-        else:
-            raise ValueError(f"Unknown print type: {print_type}")
-
-    def serialize(self, serialize_type: str) -> str:
-        if serialize_type == "json":
-            return json.dumps({"title": self.title, "content": self.content})
-        elif serialize_type == "xml":
-            root = ET.Element("book")
-            title = ET.SubElement(root, "title")
-            title.text = self.title
-            content = ET.SubElement(root, "content")
-            content.text = self.content
-            return ET.tostring(root, encoding="unicode")
-        else:
-            raise ValueError(f"Unknown serialize type: {serialize_type}")
+from app.book_implementation import Book
+from app.print_manage import ConsolePrint, ReversePrint
+from app.serializer_manage import XmlBookSerialize, JsonBookSerialize
+from app.display_manage import ConsoleDisplay, ReverseDisplay
 
 
 def main(book: Book, commands: list[tuple[str, str]]) -> None | str:
     for cmd, method_type in commands:
         if cmd == "display":
-            book.display(method_type)
-        elif cmd == "print":
-            book.print_book(method_type)
+            if method_type == "reverse":
+                ReverseDisplay.display(book)
+            elif method_type == "console":
+                ConsoleDisplay.display(book)
+            else:
+                raise ValueError(f"Unknown display type: {method_type}")
+        if cmd == "print":
+            if method_type == "reverse":
+                ReversePrint.print_book(book)
+            elif method_type == "console":
+                ConsolePrint.print_book(book)
+            else:
+                raise ValueError(f"Unknown print type: {method_type}")
+
         elif cmd == "serialize":
-            return book.serialize(method_type)
+            if method_type == "json":
+                return JsonBookSerialize.serialize_book(book)
+            elif method_type == "xml":
+                return XmlBookSerialize.serialize_book(book)
+            else:
+                raise ValueError(f"Unknown serialize type: {method_type}")
```

Comment:
You can make this more structured, also I don't see what will happen if command is not correct.